tests/executors/dask_executor.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import logging
+import unittest
+
+from airflow import configuration
+from airflow.models import DagBag
+from airflow.jobs import BackfillJob
+
+try:
+    from airflow.executors import DaskExecutor
+    from distributed import LocalCluster
+    DASK = True
+except ImportError:
+    DASK = False
+
+
+DEFAULT_DATE = datetime.datetime(2017, 1, 1)
+
+
+if DASK:
+
+    class DaskExecutorTest(unittest.TestCase):
```

Comment:
Added tests for both the Executor functions and also the process of manually stepping through running TaskInstances (simulating a job)